modules/rosa-sts-about-ocm-role.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// rosa_planning/rosa-sts-ocm-role.adoc
+//
+:_content-type: CONCEPT
+[id="rosa-sts-about-ocm-role_{context}"]
+= About the ocm-role IAM role
+
+You need to create the `ocm-role` IAM resource to enable a Red Hat organization of users to create ROSA clusters.
+
+Some considerations for your `ocm-role` IAM resource are:
+
+* You have only one `ocm-role` per Red Hat organization.
+* Any user in a Red Hat organization may create and link an `ocm-role` IAM resource.
+* Only the Red Hat organization administrator can unlink an `ocm-role` IAM resource. This limitation is to protect other Red Hat organization members from disturbing the interface capabilities of other users.
```

Comment:
Can we please capitalize Organization Administrator here (and in other topics, also saw a few mentions of 'organization admin') to be consistent with the term in other Red Hat apps/account resources?